config.example.wikimedia.yaml

```diff
@@ -120,7 +125,7 @@ spec_root: &spec_root
     /{domain:www.wikidata.org}: *wikidata.org
 
     # example for non-wikipedia
-    /{domain:ru.wikivoyage.org}: *default_project
+    /{domain:en.wikivoyage.beta.wmflabs.org}: *wikivoyage_project
```

Comment:
Why beta?

Reply:
For testing. We're talking to beta MCS and beta Parsoid, prod wikis (except en.wiki) are not set up. for Parsoid/RB in beta.

Reply:
Are we going to add prod later as well?

Reply:
Here -  no. It's just an example config, it's not used in prod. Prod config is in deploy repo.

Reply:
ok